### DISCUSSION TITLE
micro-SD card max size

### DISCUSSION THREAD

**Discussion by mfaridbadr** [erichelgeson/BlueSCSI-v1#243] [Q&A]
HI,
Kindly, i need to know what is the maximum size can i use for the microSD card  for the bluescsi  ver. 1
Example can i use 
memory card size 32G , 64 GB , ...etc.

Thanks

**Comment by erichelgeson** [comment]
2TB would be the max.

**Reply by mfaridbadr** [reply]
Thanks
